src/components/resource-section/resource-section/ResourceSection.js

```diff
@@ -51,12 +50,7 @@ const ResourceSection = ({ resourceId }) => {
     toast: toast,
   };
 
-  return (
-    <>
-      <ResourcePanel resource={resource} formOptions={formOptions} />
-      <EvaluationList evaluationsData={evaluations} />
```

Comment:
Couldn't EvaluationList component be added inside ResourceSidebar?

Reply:
It will be added on another PR together with a reactive Card component instead of a modal Sidebar.

Reply:
Ok